src/handlers/userStore.ts

```diff
@@ -0,0 +1,124 @@
+import { Subscription } from "nats";
+
+import { jsonCodec, NatsHandler } from "../nats";
+import { getClient } from "../elasticSearch";
+import { ELASTICSEARCH_INDEX_NAME } from "../config";
+import { JSONUser } from "../user";
+
+export const userStoreHandlers: NatsHandler[] = [
+    [
+        "item-store_add_user",
```

Comment:
following the new convention, this should be `add_user`. The `item-store` prefix will be added when we register the handlers